cve_bin_tool/output_engine/util.py

```diff
@@ -154,9 +157,9 @@ def format_output(
             if detailed:
                 details["description"] = cve.description
             if affected_versions != 0:
-                try:
+                if all_cve_version_info is not None:
```

Comment:
If you replace the KeyError try/except you'll need to also check that `cve.cve_number` is a valid key for `all_cve_version_info` in another way.